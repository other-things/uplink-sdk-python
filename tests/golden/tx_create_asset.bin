00000000: 00 01 00 00 00 04 74 65  73 74 00 00 00 00 00 00  ......test......
00000010: 00 00 03 E8 00 01 00 05  54 6F 6B 65 6E 00 08 44  ........Token..D
00000020: 69 73 63 72 65 74 65 00  02 00 07 63 6F 6D 70 61  iscrete....compa
00000030: 6E 79 00 0C 41 64 6A 6F  69 6E 74 20 49 6E 63 2E  ny..Adjoint Inc.
00000040: 00 06 70 6F 6C 69 63 79  00 07 73 70 65 63 69 61  ..policy..specia
00000050: 6C                                                l