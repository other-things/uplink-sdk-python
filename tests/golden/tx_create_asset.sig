r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 31020398950513132468773345845532706567089153613877695029757772072508598386779
