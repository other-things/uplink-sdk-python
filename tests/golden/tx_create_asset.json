{
    "header": {
        "contents": {
            "contents": {
                "assetName": "test",
                "assetType": {
                    "contents": null,
                    "tag": "Discrete"
                },
                "issuer": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "metadata": {
                    "company": "Adjoint Inc.",
                    "policy": "special"
                },
                "reference": "Token",
                "supply": {
                    "decimalIntegerValue": 1000,
                    "decimalPlaces": 0,
                    "tag": "Dec"
                }
            },
            "tag": "CreateAsset"
        },
        "tag": "TxAsset"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zMTAyMDM5ODk1MDUxMzEzMjQ2ODc3MzM0NTg0NTUzMjcwNjU2NzA4OTE1MzYxMzg3NzY5NTAyOTc1Nzc3MjA3MjUwODU5ODM4Njc3OQ=="
}